docs/source/yaml/location_schema.rst

```diff
@@ -12,9 +12,26 @@ The generic location schema, where ``<angle brackets>`` are placeholders, is:
        type: <footprint_type>
        <property>: <footprint_property>
      nav_poses:
-       - [<x1>, <y1>, <z1>, <yaw1>]
-       - ...
-       - [<xN>, <yN>, <zN>, <yawN>]
+       - position:
+           x: <x1>
+           y: <y1>
+         rotation_eul:
+           yaw: <yaw1>
+           angle_units: "degrees"
+       - position:
+           x: <x2>
+           y: <y2>
+         rotation_quat:
+           w: <w1>
```

Comment:
These should be `w2` and `z2` as it's the second point, yeah?

Also, just for clarity maybe include all 4 elements w x y z

Reply:
Ahh, very stupid mistake.